Source/WindowsServicify/WindowsServicify.Domain/ProcessManager.cs

```diff
@@ -40,7 +40,13 @@ private void StartProcess()
         process.StartInfo.FileName = _command;
         process.StartInfo.Arguments = _arguments;
         process.StartInfo.WorkingDirectory = _workingDirectory;
+        process.StartInfo.RedirectStandardOutput = true;
         process.Start();
+        while (!process.StandardOutput.EndOfStream)
+        {
+            string line = process.StandardOutput.ReadLine();
+            Logger.Log(line);
```

Comment:
Why should you do that?

Reply:
Well that was the only way I saw to log everything a script logs out while its being executed. I could have just used ReadToEnd but this will wait till the script finishes it's execution and does not make sense if we have a script that should run for hours and print continuously something to the console